addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,53 +4,62 @@
+                            Schedule activity
+                        </button>
                     </t>
-                </button>
-                <div class="o-autogrow"/>
-                <t t-if="chatter.hasFollowers">
-                    <div class="o_ChatterTopbar_rightSection">
-                        <div class="o_ChatterTopbar_followers">
-                            <button class="btn btn-link o_ChatterTopbar_buttonFollow" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollow">
-                                Follow
-                            </button>
-                            <button class="btn btn-link o_ChatterTopbar_buttonFollowers" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollowers">
-                                <i class="fa fa-user"/>
-                                <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonFollowersCount pl-1">0</span>
-                            </button>
+                    <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickAttachments">
+                        <i class="fa fa-paperclip"/>
+                        <t t-if="chatter.isDisabled or chatter.thread and chatter.thread.areAttachmentsLoaded">
+                            <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                        </t>
+                        <t t-else="">
+                            <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin"/>
+                        </t>
+                    </button>
+                    <div class="o-autogrow"/>
+                    <t t-if="chatter.hasFollowers">
+                        <div class="o_ChatterTopbar_rightSection">
+                                <div class="o_ChatterTopbar_followers">
+                                    <button class="btn btn-link o_ChatterTopbar_buttonFollow" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollow">
+                                        Follow
+                                    </button>
+                                    <button class="btn btn-link o_ChatterTopbar_buttonFollowers" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollowers">
+                                        <i class="fa fa-user"/>
+                                        <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonFollowersCount pl-1">0</span>
+                                    </button>
+                                </div>
                         </div>
+                    </t>
+                </div>
+                <t t-if="props.hasCloseButton">
+                    <div class="o_ChatterTopbar_buttonClose" title="Close" aria-hidden="true">
+                        <i class="fa fa-times"/>
```

Comment:
aria hidden should be on this tag

Reply:
Yes, the reasoning is that `i` is normally italics (stupid hack of font icons), and we don't want the screenreader to think there is italics here, so we say to hide it. Then the information has to be given another way. In this case the "title" would work (this could also be a sibling tag of `i` with the class `sr-only` containing the replacement text, or an aria-label on the div instead of title if we didn't care about the hover text for mouse users).